doc/source/user_guide/temperature.rst

```diff
@@ -1,8 +1,8 @@
 .. _temperature:
 
-===================================
-Temperature & Change of Temperature
-===================================
+==================================
+Temperature and temperature change
+==================================
 
```

Comment:
Please could you update units that have the first letter capitalised? There are several below (Kelvin -> kelvin). It looks really gauche 🤣